modular_zubbers/code/modules/customization/species/proteans/_proteans_species.dm

```diff
@@ -103,8 +104,26 @@
 	)
 	add_verb(gainer, protean_verbs)
 
+/datum/species/protean/proc/organ_reject(mob/living/source, obj/item/organ)
+	SIGNAL_HANDLER
+
+	if(isnull(owner))
+		return
+	var/obj/item/organ/insert_organ = organ
+	var/list/slots = list(ORGAN_SLOT_BRAIN, ORGAN_SLOT_HEART, ORGAN_SLOT_STOMACH, ORGAN_SLOT_EYES)
+	if(!(insert_organ.slot in slots))
+		return
+	if(insert_organ.organ_flags & (ORGAN_ROBOTIC | ORGAN_NANOMACHINE | ORGAN_UNREMOVABLE))
+		return
+	insert_organ.Remove(source)
+	insert_organ.forceMove(get_turf(source))
+	to_chat(source, span_danger("Your mass rejected [insert_organ]!"))
+	insert_organ.balloon_alert_to_viewers("rejected!", vision_distance = 1)
```

Comment:
is vision 1 anyone in the same tile or everyone around it? Should probably let any surgeon doing it see it too